### PR TITLE
Error string extraction

### PR DESCRIPTION
**Pull request by fgalan**
Note that as cited in the code itself (https://github.com/telefonicaid/fiware-orion/compare/hardening/error_string?expand=1#diff-c2640173c861df9babf636b17a6b3755R40) this PR doesn't do an exhaustive extraction. However, it puts some order in some existing and quite common error situations, as you can see in the .test modifications (only in error description texts).

### CODE REVIEW

File: test/functionalTest/cases/0970_create_entity/errors_at_creation.test [hunk 1]
```diff
@@ -151,27 +151,27 @@ echo
...
 HTTP/1.1 400 Bad Request
-Content-Length: 61
+Content-Length: 83
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "no entity id specified", 
+    "description": "entity id length: 0, min length supported: 1",
```
Comment (kzangeli) [inline test/functionalTest/cases/0970_create_entity/errors_at_creation.test]: Not that it is **really** important but here we lose some information.
Non-present and present-but-empty now give the same error description.

**Comment by crbrox** [comment]
LGTM

File: test/functionalTest/cases/0970_create_entity/errors_at_creation.test (hunk 1 above)
Comment (kzangeli) [inline test/functionalTest/cases/0970_create_entity/errors_at_creation.test]: Sure, NTC

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**